airflow/models/dag.py

```diff
@@ -533,94 +537,53 @@ def next_dagrun_info(
             "automated" DagRuns for this dag (scheduled or backfill, but not
             manual)
         """
-        if (
-            self.schedule_interval == "@once" and date_last_automated_dagrun
-        ) or self.schedule_interval is None:
-            # Manual trigger, or already created the run for @once, can short circuit
+        # XXX: The timezone.coerce_datetime calls in this function should not
```

Comment:
What does the `XXX` means?

Reply:
It means something along the lines of “I don’t like this but there’s no better way to do it now without digging deep into the rabbit hole and add a bunch of unrelated diffs to this PR, this should be refactored away later”. I see other projects use markers like `TODO` or `FIXME` for this as well.

https://www.snellman.net/blog/archive/2017-04-17-xxx-fixme/

Reply:
```suggestion
        # FIXME: The timezone.coerce_datetime calls in this function should not
```

We use both `TODO` and `FIXME` -- either is fine for easier `grep` :)